dags/firefox_public_data_report.py

```diff
@@ -82,4 +83,60 @@
     )
 )
 
-wait_for_main_ping >> hardware_report
\ No newline at end of file
+wait_for_clients_last_seen = ExternalTaskSensor(
+    task_id="wait_for_clients_last_seen",
+    external_dag_id="main_summary",
+    external_task_id="clients_last_seen",
+    execution_delta=timedelta(days=-6),
```

Comment:
Offhand, this is surprising

Reply:
I'm always confused about this, hence the comment on line 40.

Reply:
Got it, that makes sense now